apps/expo/src/app/tracker/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { useRef } from 'react'
+import { SafeAreaView } from 'react-native'
+import { useAuth, useUser } from '@clerk/clerk-expo'
+
+import WorkoutTracker from '~/components/tracker/workout-tracker'
+import Button from '~/components/ui/button/button'
+import CustomBottomSheetModal, {
+  CustomBottomSheetModalRef,
+} from '~/components/ui/custom-bottom-sheet-modal'
+import { useGlobalContext } from '~/context/global-context'
+
+const Tracker = () => {
```

Comment:
Onething I might worry is when integrate with the bottom nav and header, they might have some position issue (for mine, gonna do flex-1) this might fall under the same shennanigans